cardano-cli/src/Cardano/CLI/Types/Errors/TxCmdError.hs

```diff
@@ -20,6 +20,7 @@ import qualified Cardano.Api.Ledger as L
 import           Cardano.Api.Shelley
 
 import           Cardano.CLI.EraBased.Script.Mint.Types
+import           Cardano.CLI.EraBased.Script.Spend.Read
```

Comment:
This import means that the errors should go to their own separate module. This could potentially lead to annoying import cycles down the road.